cacheops/query.py

```diff
@@ -247,27 +261,72 @@ def iterator(self):
         # TODO: do not cache empty queries in Django 1.6
         superiter = self._no_monkey.iterator
         cache_this = self._cacheprofile and 'fetch' in self._cacheconf['ops']
-
+        cache_for_read = (
```

Comment:
Please rename `cache_for_read` to `should_read_cache`. Also, don't break condition into 2 lines, even flakes don't like it :), see Travis logs.